sdk/triggers.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { getTriggersListUrl, getSpecificTriggerUrl } = require('./url');
+const { getValidatedBody } = require('./handleError');
+
+async function upsert(apiKey, triggerName, triggerOptions) {
+  const listOptions = {
+    url: getSpecificTriggerUrl(triggerName),
+    headers: {
+      'X-Binaris-Api-Key': apiKey,
+    },
+    json: true,
+    body: triggerOptions,
+  };
+  return getValidatedBody(listOptions);
+}
+
+async function remove(apiKey, triggerName) {
+  const listOptions = {
```

Comment:
They are all called `listOptions` even for `remove` and `upsert`.

Reply:
changed all of them to just `options`